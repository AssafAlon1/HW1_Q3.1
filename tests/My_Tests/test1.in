1 1 2 2 6 6 7 7 8 8 9 9 10 10 3 3 4 4 5 5
1 -1 3 -3 5 -5 7 -7 9 -9
0