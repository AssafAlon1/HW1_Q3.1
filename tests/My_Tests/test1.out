Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
Enter input for data
Enter input for key
item #1
key:  1
dat:  -1

item #2
key:  2
dat:  2

item #3
key:  3
dat:  -3

item #4
key:  4
dat:  4

item #5
key:  5
dat:  -5

item #6
key:  6
dat:  6

item #7
key:  7
dat:  -7

item #8
key:  8
dat:  8

item #9
key:  9
dat:  -9

item #10
key:  10
dat:  10

